README.md

```diff
@@ -57,6 +57,22 @@ build gossamer command:
 ```
 make gossamer
 ```
+### Troubleshooting for Apple Silicon users
+
+If you are facing the following problem with the `wasmer`:
+````
+undefined: cWasmerImportObjectT
+undefined: cWasmerImportFuncT
+undefined: cWasmerValueTag
+````
```

Comment:
Can you change that to use 3 backticks instead of 4? Sorry I can't do a commit suggestion due to those backticks haha

Reply:
Done!